src/flags.rs

```diff
@@ -112,6 +111,11 @@ impl Flags {
             _ => vec![Block::Name],
         };
 
+        // Add inode as first column if with inode flag
```

Comment:
If `inode` is in blocks then do not change its position.

We just have to check if we have `-l` and `-i` without `--blocks` and in that case have inode infront.
The change should be going somewhere here: https://github.com/Peltoche/lsd/blob/87e87d87cdf91d7e8a3f385b0a5beb96ddbd8824/src/flags.rs#L99

Reply:
I have checked is the `blocks` containing the `inode` below this.

I update the blocks below the creation is because there are too many cases if combining into one `if else`

- `-l -i`
- `-i --blocks name,inode`
- `-i --blocks name`
- `-l -i --blocks name,inode`

So I check is `blocks` containing inode, and is `inode flag` set, and update `blocks` if needed

Reply:
My bad, I thought the code did something else.
This looks good.

Reply:
Sorry for interrupting your vacation, have fun for it!😁